src/cf-core/src/cf-base.less

```diff
@@ -103,12 +102,14 @@ body {
 
 .heading-5( @fs: @size-v )  {
     @font-size: @fs;
-    .u-webfont-demi();
+    .u-webfont-medium();
 
     margin-bottom: unit( 15px / @font-size, em );
     font-size: unit( @font-size / @base-font-size-px, em );
+    font-weight: bold;
     letter-spacing: 1px;
     line-height: 1.25;
+    text-shadow: 0px 0px @text;
```

Comment:
Can we move the color to a param so users don't have to know that they need to over-write this when they change the text color?

Reply:
I thought about this, I'll give it a try and set the default value to @text. Thanks for the nudge!